tsdb/aggregate.go

```diff
@@ -381,8 +381,18 @@ func (e *AggregateExecutor) processFunctions(results [][]interface{}, columnName
 	callInPosition := e.stmt.FunctionCallsByPosition()
 	hasTimeField := e.stmt.HasTimeFieldSpecified()
 
+	FlatCallInPositions := make([][]*influxql.Call, 0)
```

Comment:
Comment here might help clarify for the next person why you need this.  I understand it, but I suspect this is confusing to people seeing it for the first time.